app/views/announcement/_new.html.erb

```diff
@@ -1,16 +1,21 @@
 <%= form_for @announcement, url: {action: "create"} do |f| %>
```

Comment:
This form goes off the page. Cannot scroll to reach bottom of form, only way to submit is to zoom out of page
![screen shot 2017-12-01 at 2 27 41 pm](https://user-images.githubusercontent.com/26726721/33501883-ce1b1f2a-d6a3-11e7-9587-af41301ef7b4.png)

Reply:
For a separate PR – the modals need to update for different screen sizes